integration-tests/applications/src/test/java/com/xsk/integration/tests/applications/deployment/client/PublisherClient.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 SAP SE or an SAP affiliate company and XSK contributors
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Apache License, v2.0
+ * which accompanies this distribution, and is available at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and XSK contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package com.xsk.integration.tests.applications.deployment.client;
+
+import com.xsk.integration.tests.applications.deployment.DeploymentException;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.methods.HttpUriRequest;
+import org.apache.http.client.methods.RequestBuilder;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.concurrent.CompletableFuture;
+
+import static com.xsk.integration.tests.applications.deployment.ApplicationDeploymentRule.HOST;
+
+public class PublisherClient {
+    private static final String PUBLISHER_SERVICE_URL = HOST + "/services/v4/ide/publisher/request/";
```

Comment:
Couldn't `HOST` be an `URI`? If it can, we can use its `resolve` methods and others instead of concatenating strings that create an `URI`. Concatenation may cause tricky errors as it depends on the developer to set `/` where necessary